features/src/main/scala/com/salesforce/op/utils/spark/OpVectorMetadata.scala

```diff
@@ -72,6 +75,8 @@ class OpVectorMetadata private
     newColumns: Array[OpVectorColumnMetadata]
   ): OpVectorMetadata = OpVectorMetadata(name, newColumns, history)
 
+  val textTypes = Seq(MultiPickList, MultiPickListMap, Text, TextArea, TextAreaMap, TextMap, Binary, BinaryMap,
```

Comment:
1. how is `Binary` and `BinaryMap` are testTypes?
2. also instead please use `FeatureType.shortTypeName[Text]` etc

Reply:
I want the package name as well. E.g `com.salesforce.features.types. MultiPickList`

As it is specified in `OpVectorColumnMetadata.parentFeatureType`

Reply:
Then FeatureType.typeName